java/trace/src/main/scala/org/typelevel/otel4s/java/trace/SpanContextConversion.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.otel4s.java.trace
+
+import io.opentelemetry.api.trace.{SpanContext => JSpanContext}
+import io.opentelemetry.api.trace.{TraceFlags => JTraceFlags}
+import io.opentelemetry.api.trace.{TraceState => JTraceState}
+import org.typelevel.otel4s.trace.SpanContext
+import org.typelevel.otel4s.trace.TraceFlags
+import org.typelevel.otel4s.trace.TraceState
+import scodec.bits.ByteVector
+
+private[otel4s] object SpanContextConversion {
+
+  def fromJSpanContext(context: JSpanContext): SpanContext = {
```

Comment:
thoughts on `toScala` for this one and `toJava` for the other? a bit shorter name-wise and consistent with what the stdlib does for its conversions.

how often will end users need to use these conversions? if it's not super rare, extension methods would probably be a nice touch, though I'm happy to handle that if it ends up being needed

Reply:
Much better than `fromJSpanContext` and `toJSpanContext`, thanks!